appcues/src/main/java/com/appcues/trait/AppcuesTraitException.kt

```diff
@@ -3,4 +3,13 @@ package com.appcues.trait
 /**
  * A type of exception that can occur during the application of an ExperienceTrait.
  */
-public class AppcuesTraitException(message: String) : Exception(message)
+public class AppcuesTraitException(
+    message: String,
+
+    /**
+     *  When set, this value can indicate a number of milliseconds after which re-application
+     *  of traits can be attempted, to try to auto-recover from this error.
+     */
+    public val retryMilliseconds: Int? = null,
```

Comment:
fyi, there is some discussion on the iOS side over whether we want to expose this value publicly https://github.com/appcues/appcues-ios-sdk/pull/434#discussion_r1287648945